### ISSUE TITLE
Accessibility: Module/provider list should be formatted as a list

### ISSUE THREAD

**Issue by ghost** [opentofu/registry-ui#159]
Currently, the module list at `/modules` and the provider list at `/providers` is formatted with headings. For more efficient navigation, it would be good if these items were a list.
